src/api/routes/myGroups.ts

```diff
@@ -45,41 +57,44 @@ const myGroups = router({
       return "/fakeMeeting";
     }
 
-    if (group.meetingLink && !meetingLinkIsExpired(group.updatedAt)) {
-      return group.meetingLink;
-    }
-
-    const now = Math.floor(Date.now() / 1000);
-    const groupName = formatGroupName(group.name, group.groupUsers.length);
-    const res = await createMeeting(encodeMeetingSubject(group.id, groupName), now, now + 3600);
-    invariant(res.meeting_info_list.length === 1);
-
-    const meetingLink = res.meeting_info_list[0].join_url;
-
-    await Group.update({
-      meetingLink,
-    }, {
-      where: {
-        id: group.id,
+      // if the user's group is present in OngoingMeetings Model, return the meeting link 
+      const ongoingMeeting = await OngoingMeetings.findOne({ where: { groupId: group.id } });
+      if (ongoingMeeting) {
+        return ongoingMeeting.meetingLink;
       }
-    });
+      // if not shown in OngoingMeetings, check if there are empty slots to join
+      // find TM user ids that are not in use and generate a meeting link using an available id
+      const TM_USER_IDS = apiEnv.TM_ADMIN_USER_IDS.split(',');
```

Comment:
As I mentioned elsewhere, don't split here. TM_ADMIN_USER_IDS should already be a string array.